src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -637,6 +637,8 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addCheckbox("Combine multiple bit tips into one", s.stackBits);
     layout.addCheckbox("Ask for confirmation when uploading an image",
                        s.askOnImageUpload);
+    layout.addCheckbox("Show a popup on hiding tabs",
+                       s.informOnTabVisibilityToggle);
```

Comment:
I'm wondering whether it really makes sense to add a setting for this. Showing the popup on first toggle is good IMO. After clicking "Don't show again", I don't think most users would like to re-enable the popup at some point in the future.
Curious to hear more opinions on this though

Reply:
This was more of a setting for testing, however I just went with what we already had for a similar popup - the one that you see upon trying to paste the image to Chatterino.

![mm added this lole](https://cdn.zneix.eu/ocn8sLL.png)

Maybe we should get rid of this as well? No idea, just wanted to be consistent

Reply:
I think the use case of that popup is slightly different -- it gives you the ability to cancel an upload. In this case, the popup just informs the user of the action they are about to make.